src/ansi_color.go

```diff
@@ -46,7 +45,7 @@ func getColorFromName(colorName string, isBackground bool) (string, error) {
 
 // AnsiColor writes colorized strings
 type AnsiColor struct {
-	buffer  *bytes.Buffer
+	builder strings.Builder
```

Comment:
Noticeable?

Reply:
Inwas reading through string concatenation performance benchmarks and this one came out on top. So it made sense to swap it. They made it compatible with every other way to ensure an easy transition.